nav2_bringup/params/nav2_params.yaml

```diff
@@ -96,6 +96,7 @@ controller_server:
     progress_checker_plugins: ["progress_checker"]
     goal_checker_plugins: ["general_goal_checker"] # "precise_goal_checker"
     controller_plugins: ["FollowPath"]
+    path_handler_plugins: ["path_handler"]
```

Comment:
To just save a hassle in the other ticket on renaming these, we can rename this now to `PathHandler`